integration.md

```diff
@@ -0,0 +1,104 @@
+## Integration
+- The cloudly-storage package needs to be added in the dependencies of the package.json file and needs to be installed.
+- A Key Value store needs to be created on the cloudflare account used by the worker in order to use this database in the cloudflare worker.
+- The durable object class Backend needs to be exported from the main ```index.ts``` file in your worker:
+```ts
+export { Backend } from "cloudly-storage"
+
+export default {
+	async fetch(
+	...
+```
+
+### wrangler.toml
+The wrangler.toml file needs to include a Key Value store and a durable object binding:
+```
+kv-namespaces = [
+			{ binding = "archive", id = "id_found_in_the_cloudflare_dashboard", preview_id = "id_found_in_the_cloudflare_dashboard" }
+		[
+[durable_objects]
+bindings = [
+		{ name = "BufferBackend", class_name = "Backend" }
+	   [
+```
+The binding in the kv-namespaces and the name of the durable_objects bindings can be whatever you want.
+
+### Opening the DB
+Lets say we want to open a transactional database that can store users with an interface User.
+```ts
+import { User } from "../model/User"
+import * as gracely from "gracely"
+
+type Layout = { collection: { user: User } }
```

Comment:
How do we decide how the Layout type should look like? Is it always an object with a collection prop or is that just an example?

Reply:
The layout can be different, should I link to the type in the documentation?

Reply:
Yes that would be good. Makes it easier to know how to write the layout :)